package.json

```diff
@@ -22,31 +22,34 @@
   "repository": "github:SignalK/instrumentpanel.git",
   "scripts": {
     "start": "node server.js",
-    "build": "webpack -p",
-    "build-debug": "webpack -p --devtool source-map",
+    "build": "webpack -p --progress",
+    "build-debug": "webpack -d --devtool cheap-module-source-map --progress",
     "watch": "webpack -d --watch",
     "changelog": "github-changes -o signalk -r instrumentpanel -a --only-pulls --use-commit-body --data=pulls --tag-name=v$npm_package_version",
     "release": "git tag -d v$npm_package_version ; npm run changelog && git add CHANGELOG.md && git commit -m 'chore: update changelog' && git tag v$npm_package_version && git push --tags && git push && git tag -d latest && git push origin :refs/tags/latest && git tag latest && git push origin tag latest"
   },
   "devDependencies": {
-    "@signalk/client": "^0.1.4",
-    "babel": "^5.8.23",
-    "babel-loader": "^5.3.2",
+    "@signalk/client": "^0.2.1",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
     "bacon.model": "^0.1.12",
-    "baconjs": "^0.7.42",
-    "bluebird": "^2.9.24",
-    "d3": "^3.5.6",
-    "debug": "^2.1.1",
-    "github-changes": "^1.0.4",
+    "baconjs": "^0.7.28",
```

Comment:
Something weird going on here.

Reply:
I found. 0.7.28 version is the minimal version dependencie for bacon.model. Bad copy/paste... Fixed with the latest version 2.0.10